tools/topology/topology2/include/pipelines/cavs/deepbuffer-playback.conf

```diff
@@ -49,6 +49,7 @@ Class.Pipeline."deepbuffer-playback" {
 			copier_type	"host"
 			type	"aif_in"
 			node_type $HDA_HOST_OUTPUT_CLASS
+
```

Comment:
spurious change

Reply:
no, this is intentional. 

this was to make sure all uses of the copier_type, type and node_type are well isolated to make sure we can detect any differences.